Readme.md

```diff
@@ -430,6 +438,34 @@ Division by zero is not supported.
 1337
 ```
 
+##### Customizing Error Reporting
```

Comment:
I think this could just be added to the previous part ("Reporting errors") instead as an addition (i.e. "you can also choose to show help text, bla bla bla").

Reply:
Sounds good. I'll change.